source/wp-content/themes/wporg-developer-2023/src/code-title/block.php

```diff
@@ -24,14 +24,33 @@ function init() {
 /**
  * Render the block content.
  *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
  * @return string Returns the block markup.
  */
-function render( $attributes ) {
+function render( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	$post_ID = $block->context['postId'];
+	$content = get_signature( $post_ID );
```

Comment:
Any idea on how we will be able to render this dynamic block inside another dynamic block with the right context? In the hooks component, we'll need to do that. We'll need to loop through the hooks and construct a couple of these blocks.

I can't seem to find a way to retrieve a string of a _rendered_ dynamic block without using `do_blocks` which outputs the block to the document. 

My only other thought is to pass the `postId` through.